cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -345,6 +345,16 @@ install() {
 {% else %}
     {{ error("unimplemented install_path format: " ~ install_path.kind) }}
 {% endif %}
+    # ...ignoring all of the above, if the user asked us to completely override
+    # those choices and use a specified directory, then pick that now
+    if [ -n "${CARGO_DIST_FORCE_INSTALL_DIR:-}" ]; then
+        _install_home="$CARGO_DIST_FORCE_INSTALL_DIR"
+        _install_dir="$CARGO_DIST_FORCE_INSTALL_DIR/bin"
+        _env_script_path="$CARGO_DIST_FORCE_INSTALL_DIR/env"
+        _install_dir_expr='$CARGO_DIST_FORCE_INSTALL_DIR/bin'
+        _env_script_path_expr='$CARGO_DIST_FORCE_INSTALL_DIR/env'
+    fi
```

Comment:
the powershell code seems to be carefully making sure the install receipt gets the right value, but not so for shell? ` RECEIPT=$(echo "$RECEIPT" | sed "s,AXO_CARGO_HOME,$_install_dir,")` occurs much higher up...

Reply:
Good call, updated.